src/supermarket/app/controllers/collaborators_controller.rb

```diff
@@ -93,9 +93,9 @@ def destroy_group
                             "#{collaborator.user.username} is still a collaborator on this #{params[:resourceable_type]}" " "
                           end
       end
-
+      resource_path_str = "#{params[:resourceable_type].underscore}_path"
       redirect_to(
-        resource
+        send(resource_path_str, resource)
```

Comment:
what is the need for extra variables `resource_path_str`

Reply:
just for the readibility.